waiter/src/waiter/core.clj

```diff
@@ -915,6 +921,49 @@
                                                              determine-priority-fn ws/process-response! ws/process-exception-in-request
                                                              ws/abort-request-callback-factory request))]
                                          (ws/request-handler password process-request-fn request)))))
+   :display-kv-store-state-handler-fn (pc/fnk [[:curator kv-store]
```

Comment:
Can we remove `display-` from all these handler names?

Reply:
Done.